README.md

```diff
@@ -90,6 +90,10 @@ The driver packs form the base of the device images. Each version of JetPack is
 
 Note: not all combinations are valid and the `Makefile` should have all valid combinations declared.
 Note: if these command's are not run on the device, the `DOCKER_HOST` variable must be set.
+Note: from 4.2, cross-device building is supported: leave the `DOCKER_HOST` empty and do this once before running make on the host device:
```

Comment:
I don't think this is needed as qemu-aarch64-static is in the base l4t image now.

Reply:
Referring to this particular comments, I think one still have to do the qemu registration on the host machine (if it was never done before or the host never had QEMU installed). The QEMU static bin was copied (by docker service) to l4t arm64v8 base image. I think during build, it relies on the host kernel's binfmt registration to correctly load QEMU as the intepretor when first RUN is issued during building of l4t image, e.g. from line 10 and on of https://github.com/direbearform/jetson-containers/blob/master/docker/l4t/32.1/nano/Dockerfile. What do you think?

Reply:
You are correct. The host has to have `binfmt-support` installed or you'll get `exec user process caused "exec format error"` in the build.

Telling the user to run a docker container with privilege to do things scares me. I think it is better to tell the user to install `qemu-user`/`qemu-user-static` which will install `binfmt-support` allowing everything to work.

When building on the device, QEMU will be installed in the container (but not active) having no `binfmt-support` in the host OS.

Reply:
`sudo apt-get update && sudo apt-get install -y --no-install-recommends qemu-user-static binfmt-support` would be preferred to running privileged container IMHO. Thoughts?

Reply:
The only for that commend is also the support for Windows Docker CE (where you don't get to do this). But I am fine with what you suggested too since building from Windows Docker is not a main supported scenario in your case anyway.